src/Markdig/Extensions/Emoji/EmojiMapping.cs

```diff
@@ -0,0 +1,1784 @@
+                {":norfolk_island:","🇳🇫"},
+                {":nigeria:","🇳🇬"},
+                {":nicaragua:","🇳🇮"},
+                {":netherlands:","🇳🇱"},
+                {":norway:","🇳🇴"},
+                {":nepal:","🇳🇵"},
+                {":nauru:","🇳🇷"},
+                {":niue:","🇳🇺"},
+                {":new_zealand:","🇳🇿"},
+                {":oman:","🇴🇲"},
+                {":panama:","🇵🇦"},
+                {":peru:","🇵🇪"},
+                {":french_polynesia:","🇵🇫"},
+                {":papua_new_guinea:","🇵🇬"},
+                {":philippines:","🇵🇭"},
+                {":pakistan:","🇵🇰"},
+                {":poland:","🇵🇱"},
+                {":st_pierre_miquelon:","🇵🇲"},
+                {":pitcairn_islands:","🇵🇳"},
+                {":puerto_rico:","🇵🇷"},
+                {":palestinian_territories:","🇵🇸"},
+                {":portugal:","🇵🇹"},
+                {":palau:","🇵🇼"},
+                {":paraguay:","🇵🇾"},
+                {":qatar:","🇶🇦"},
+                {":reunion:","🇷🇪"},
+                {":romania:","🇷🇴"},
+                {":serbia:","🇷🇸"},
+                {":rwanda:","🇷🇼"},
+                {":saudi_arabia:","🇸🇦"},
+                {":solomon_islands:","🇸🇧"},
+                {":seychelles:","🇸🇨"},
+                {":sudan:","🇸🇩"},
+                {":sweden:","🇸🇪"},
+                {":singapore:","🇸🇬"},
+                {":st_helena:","🇸🇭"},
+                {":slovenia:","🇸🇮"},
+                {":svalbard_jan_mayen:","🇸🇯"},
+                {":slovakia:","🇸🇰"},
+                {":sierra_leone:","🇸🇱"},
+                {":san_marino:","🇸🇲"},
+                {":senegal:","🇸🇳"},
+                {":somalia:","🇸🇴"},
+                {":suriname:","🇸🇷"},
+                {":south_sudan:","🇸🇸"},
+                {":sao_tome_principe:","🇸🇹"},
+                {":el_salvador:","🇸🇻"},
+                {":sint_maarten:","🇸🇽"},
+                {":syria:","🇸🇾"},
+                {":swaziland:","🇸🇿"},
+                {":tristan_da_cunha:","🇹🇦"},
+                {":turks_caicos_islands:","🇹🇨"},
+                {":chad:","🇹🇩"},
+                {":french_southern_territories:","🇹🇫"},
+                {":togo:","🇹🇬"},
+                {":thailand:","🇹🇭"},
+                {":tajikistan:","🇹🇯"},
+                {":tokelau:","🇹🇰"},
+                {":timor_leste:","🇹🇱"},
+                {":turkmenistan:","🇹🇲"},
+                {":tunisia:","🇹🇳"},
+                {":tonga:","🇹🇴"},
+                {":tr:","🇹🇷"},
+                {":trinidad_tobago:","🇹🇹"},
+                {":tuvalu:","🇹🇻"},
+                {":taiwan:","🇹🇼"},
+                {":tanzania:","🇹🇿"},
+                {":ukraine:","🇺🇦"},
+                {":uganda:","🇺🇬"},
+                {":us_outlying_islands:","🇺🇲"},
+                {":united_nations:","🇺🇳"},
+                {":uruguay:","🇺🇾"},
+                {":uzbekistan:","🇺🇿"},
+                {":vatican_city:","🇻🇦"},
+                {":st_vincent_grenadines:","🇻🇨"},
+         
```

Comment:
The ctor accepting a dictionary can be used directly here, otherwise the parameters should be `emojiToUnicode.Count, emojiToUnicode.Count * 2, emojiToUnicode.Count * 2` to save a few allocations on resizes.

Reply:
I didn't use the ctor here because I'm looping anyway to detect first characters. I thought that it would be more clear to also add at the same time, in the loop. What do you think?

(I will add the other parameters)

Reply:
Up to personal preference, if you find it more readable I'm fine with that as long as the parameters are updated (the ctor will choose the same numbers and foreach over it the same way)